### PR TITLE
Remove red mark from grid

### LINKED ISSUE #2092: Notifications editing issue
@Abyss777 

There is a minor issue in notifications editing dialog. It seems like items are marked as dirty because I see red triangle in top left corner.

### PR DESCRIPTION
**Pull request by Abyss777**
Fix #2092

### CODE REVIEW

File: web/app/view/Notifications.js [hunk 1]
```diff
@@ -29,6 +29,10 @@ Ext.define('Traccar.view.Notifications', {
         selType: 'cellmodel',
     },
 
+    viewConfig:{
```
Comment (tananaev) [inline web/app/view/Notifications.js]: space is missing between colon and open bracket

**Comment by tananaev** [comment]
Please add missing space.

**Comment by tananaev** [comment]
Never mind. I'll fix it.

**Comment by Abyss777** [comment]
Sorry, I going back to computer after few hours.
9 июля 2016 г. 9:59 пользователь "Anton Tananaev" <notifications@github.com>
написал:

**Pull request merged**